java-core/google-cloud-core/src/main/java/com/google/cloud/testing/BaseEmulatorHelper.java

```diff
@@ -45,6 +45,7 @@
 import java.nio.file.Path;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import java.time.Duration;
```

Comment:
Same with this.

Reply:
This is what caused 7005 (https://github.com/googleapis/sdk-platform-java/pull/3330#discussion_r1826079247)